hazelcast/src/main/java/com/hazelcast/transaction/impl/xa/XATransaction.java

```diff
@@ -175,18 +175,41 @@ public void commit() throws TransactionException, IllegalStateException {
         }
     }
 
-    public void commitAsync(ExecutionCallback callback) {
+    public void commitAsync(final ExecutionCallback callback) {
```

Comment:
Shouldn't we add these too?
https://github.com/hazelcast/hazelcast/pull/16579/files#diff-0c104d8009708b16b9092edfd2438fa1R173-R176

Reply:
I found the same is missing in https://github.com/hazelcast/hazelcast/pull/16618 and https://github.com/hazelcast/hazelcast/pull/16617.

Reply:
didn't get this?